src/cmake/dependencies.cmake

```diff
@@ -148,7 +148,7 @@ if(BUILD_WITH_BULLET AND NOT USE_SYSTEM_BULLET)
   # libs, causing linker errors on Magnum side. If you have CMake 3.13, the
   # Find module is able to correct that on its own, otherwise you need to
   # enable BUILD_SHARED_LIBS to build as shared.
-  if(NOT CORRADE_TARGET_EMSCRIPTEN)
+  if((NOT CORRADE_TARGET_EMSCRIPTEN) AND CMAKE_VERSION VERSION_LESS 3.13)
     set(BUILD_SHARED_LIBS ON CACHE BOOL "" FORCE)
   else()
     # On Emscripten we require 3.13, so there it's fine (and there we can't use
```

Comment:
Actually, we don't error anymore on CMake < 3.13. Don't we have to add a specific additional require or else people could fail on emscripten silently if build_with_bullet and < 3.13. I'd add an error if ```CORRADE_TARGET_EMSCRIPTEN AND CMAKE_VERSION VERSION_LESS 3.13```

Reply:
Don't worry, I already thought about that -- it's done by FindBullet itself ;) https://github.com/mosra/magnum-integration/blob/a897805489102030da09644f264983944ddbf824/modules/FindBullet.cmake#L73-L75

Reply:
Ah, just tested this. Good thinking.